Loading/ru.lproj/InfoPlist.strings

```diff
@@ -1 +1 @@
-CFBundleName = "Загрузка";
CFBundleDisplayName = "Загрузка";
NSHumanReadableCopyright = "© 2015 Mike McFadden / Bonzai Программы\nВсе права защищены.";
\ No newline at end of file
+CFBundleName = "Loading";
CFBundleDisplayName = "Loading";
NSHumanReadableCopyright = "© 2015 Mike McFadden / Bonzai Apps\nВсе права защищены.";
```

Comment:
Ah, so you wanted them to be translated too… I mean, I don't feel that the Russian word for «Loading» would make a good app name, but OK, I'll translate them.

Considering «Bonzai Apps»: I guess it's a company name, so it shouldn't really be translated. Am I wrong?